static/app/views/monitors/components/monitorStats.tsx

```diff
@@ -177,4 +180,10 @@ const StyledHeaderTitle = styled(HeaderTitle)`
   margin-bottom: ${space(1)};
 `;
 
+const Placeholder = styled('div')<{height: number}>`
```

Comment:
should we use the prebuilt `Placeholder` component?

Reply:
oh yes! — forgot that it existed lol